lib/beamchmark/formatters/html/templates.ex

```diff
@@ -49,6 +50,67 @@ defmodule Beamchmark.Formatters.HTML.Templates do
     }
   end
 
+  defp format_float(float) do
+    float |> Float.round(2)
+  end
+
+  @spec formatted_average_cpu_usage([CpuInfo.cpu_usage_t()]) :: %{
+          (cpu_usage_entry :: atom()) => String.t()
+        }
+  def formatted_average_cpu_usage(cpu_snapshots_reversed) do
+    cpu_snapshots = Enum.reverse(cpu_snapshots_reversed)
+
+    %{
+      average_cpu_usage:
+        Enum.map_join(cpu_snapshots, ", ", fn %{cpu_usage: _, average_all_cores: avg} ->
+          format_float(avg)
+        end),
+      time_stamps: Enum.map_join(1..(length(cpu_snapshots) - 1), ", ", fn el -> el end)
+    }
+  end
```

Comment:
```suggestion
  @spec format_average_cpu_usage([CpuInfo.cpu_usage_t()]) :: %{
          (cpu_usage_entry :: atom()) => String.t()
        }
  def format_average_cpu_usage(cpu_snapshots_reversed) do
    cpu_snapshots = Enum.reverse(cpu_snapshots_reversed)

    %{
      average_cpu_usage:
        Enum.map_join(cpu_snapshots, ", ", fn %{cpu_usage: _cpu_usage, average_all_cores: avg} ->
          format_float(avg)
        end),
      timestamps: Enum.map_join(1..(length(cpu_snapshots) - 1), ", ", fn el -> el end)
    }
  end
```

However, `timestamps` are not true timestamps. They are indicies so either change x-axis to be not in seconds or add timestamp information to cpu_snapshots? Maybe cpu_snapshots should be a list of tuples {timestamp, cpu_snapshot}?

Reply:
I think there is no need in collecting time, because we can easily calculate it having `duration` and `interval`. The timestamp needs to be renamed to more intuitive `time` to show that it represents abstract time. What do you think?

Reply:
Ok but remember that if we keep x-axis in seconds then we cannot just show snapshot indicies. 
Consider scenario in which we have `duration=30s` and `interval=3s`. In such a case we would have numbers 1..10 on x-axis while we should have 1..30

Reply:
It is never meant to be in seconds, it is an abstract time. Do you think that it needs to be in seconds?

Reply:
Ach right, I thought we have x-axis in seconds. Ok then indicies are good.